graphql/query_resolver.go

```diff
@@ -438,18 +438,16 @@ func (r *queryResolver) Project(ctx context.Context, projectIdentifier string) (
 
 // Projects is the resolver for the projects field.
 func (r *queryResolver) Projects(ctx context.Context) ([]*GroupedProjects, error) {
-	allProjects, err := model.FindAllMergedTrackedProjectRefs()
+	usr := mustHaveUser(ctx)
+	viewableProjects, err := usr.GetViewableProjectSettings(ctx)
```

Comment:
tiny nit: could we call this viewableProjectIds so it's clearer while we're also re-querying them?